### PR TITLE
Adding Jellyfin and Podcasts browse tiles

### PR DESCRIPTION
**Pull request by phampyk**
Fix Soma FM tile not showing and added Jellyfin and podcasts tiles
Background images from unsplash
Jellyfin icon from their website
Microphone icon from icons8.com

### CODE REVIEW

File: mopidy_iris/static/app.js [hunk 1]
```diff
@@ -188341,8 +188341,16 @@ var Browse = /*#__PURE__*/function (_React$Component) {
...
+              case 'SomaFM':
                 subdirectory.icons = ['/iris/assets/backgrounds/browse-somafm.jpg'];
+                break;
+				
+			  case 'Jellyfin':
+                subdirectory.icons = ['/iris/assets/backgrounds/browse-jellyfin.jpg'];
+                break;
+				
+			  case 'Podcasts':
+                subdirectory.icons = ['/iris/assets/backgrounds/browse-podcasts.jpg'];
```
Comment (jaedb) [inline mopidy_iris/static/app.js:188353]: These edits are in the compiled application code and will be overwritten next time it is compiled.

Instead, you want to editing the source file: https://github.com/jaedb/Iris/blob/master/src/js/views/Library/Browse.js#L86-L88

**Review by jaedb — CHANGES_REQUESTED** [review]
Nice work! The files you've edited are all in the _compiled_ codebase. For your contribution to be persistent, you will need to edit the _source_ files instead.

Everything you need is in the `/src` directory.

File: mopidy_iris/static/app.js (hunk 1 above)
Comment (phampyk) [reply mopidy_iris/static/app.js]: Thank you for letting me know where to do the change, I hope now everything is right.
And thank you for your work too, is very useful for me :)

**Comment by jaedb** [comment]
FYI you forgot to relocate your _images_ into the `src` folder. Something to remember for next time! Thanks for the contribution.

**Pull request merged**